packages/logging/src/in-memory-logger.ts

```diff
@@ -1,4 +1,4 @@
-import { Logger, Metadata } from "@fwl/logging";
+import { Logger, Metadata } from "./logger";
```

Comment:
This worked before because of yarn workspaces and I have no idea how it worked

Reply:
I think that I've seen something about that. I'll try to find the link